README.md

```diff
@@ -115,6 +119,50 @@ Commit and push these changes to your local clone on GitHub, and then [create a
 
 We will be notified automatically when you have created your pull request, and providing that everything adheres to nf-core guidelines we will endeavour to approve your pull request as soon as possible.
 
+## Adding a new pipeline specific config
+
```

Comment:
Explain the inheritence of configs here: i.e. base.config <  institutional.config < institutional_pipeline.config 

Where each subsequent config overwrites the same parameter in the previous config file, and new parameters are added.

Change my phrasing as you see fit.

Reply:
I see what you want here, but we're not really explaining what the instutional.config is doing in term of overwriting params.
I'll try to rework the whole file to address that.
It's definitively a good point.

Reply:
Sounds good. I wasn't sure how nf-core operates in that area - a lot of the docs appear to have a certain level of implicit knowledge whereas I prefer to be overly explicit.